src/js/IRSReport.jsx

```diff
@@ -1,14 +1,65 @@
 var React = require('react');
+var api = require('./api');
 
 var IRS = React.createClass({
-  propTypes: {
+  getInitialState: function() {
+    return {
+      irs: {
+        msas: []
+      }
+    }
+  },
 
+  componentWillMount: function() {
+    var self = this;
+    api.getIRS(function(irsObj){
+      self.setState({
+        irs: irsObj
+      });
+    });
   },
 
   render: function() {
+    var self = this;
     return (
-      <div className="IRS">
-        This is the IRS report.
+      <div className="IRSReport EditsHeaderDescription">
```

Comment:
We probably want another class here, or at least to rename the class to be more general